README.md

```diff
@@ -0,0 +1,173 @@
+topic_root + "/ExperienceMap/SetGoalPose" ([geometry_msgs/PoseStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html))
+> The goal pose to where path will be planned.
+
+#### 4.1.2 Published Topics
+
+topic_root + "/ExperienceMap/Map" (ratslam_ros/TopologicalMap)
+> The experience map in RatSLAM format.
+
+topic_root + "/ExperienceMap/MapMarker" ([visualization_msgs/Marker](http://docs.ros.org/api/visualization_msgs/html/msg/Marker.html))
+> The experience map as ROS visualization markers for displaying puposes.
+
+topic_root + "/ExperienceMap/RobotPose" ([geometry_msgs/PoseStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html))
+> The current pose of the robot in map.
+
+topic_root + "/ExperienceMap/PathToGoal" ([nav_msgs/Path](http://docs.ros.org/api/nav_msgs/html/msg/Path.html))
+> The current planned path (published if triggered to plan).
+
+#### 4.1.3 Parameters
+
+**Note:** Only ROS parameters are mentioned. All others are explained in external documentation. The parameter "topic_root" can be found there!
+
+"map_frame" (Default: "map")
+> The name of the map frame.
+
+"odom_frame" (Default: "odom")
+> The name of the odometry frame.
+
+"base_frame" (Default: "base_footprint")
+> The name of the base frame.
+
+"tf_update_rate" (Default: 20.0)
+> The update rate [hz] of the map_frame -> odom_frame transformation broadcaster.
+
+"map_save_period" (Default: 10.0)
+> The period [seconds] after which a map is saved.
+
+"map_file_path" (Default: "ratslam-latest.bmap")
+> The file path where map is saved.
+
+#### 4.1.4 Required tf Transforms
+
+odom_frame -> base_frame
+> Usually provided by the odometry system (e.g., the driver for the mobile base).
+
+#### 4.1.5 Provided tf Transforms
+
+map_frame -> odom_frame
+> The current estimate of the robot's pose within the map frame.
+
+### 4.2 ratslam_pc
+
+The pose cell network is used to track odometry information based on movement, vision and recognition.
+
+#### 4.2.1 Subscribed Topics
+
+"odom" ([nav_msgs/Odometry](http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html))
+> The current odometry.
+
+topic_root + "/LocalView/Template" (ratslam_ros/ViewTemplate)
+> The current matched template.
+
+#### 4.2.2 Published Topics
+
+topic_root + "/PoseCell/TopologicalAction" (ratslam_ros/TopologicalMap)
+> The topological action performed by network.
+
+#### 4.2.3 Parameters
+
+**Note:** Only ROS parameters are mentioned. All others are explained in external documentation.
+
+"pcn_save_period" (Default: 10.0)
+> The period [seconds] after which a pose cell network state is saved.
+
+"pcn_file_path" (Default: "ratslam-latest.bpcn")
+> The file path where pose cell network state is saved.
+
+### 4.3 ratslam_lv
+
+The local view matcher is used to match the camera templates against previous experienced ones. This node provides the matching of images.
+
+#### 4.3.1 Subscribed Topics
+
+"image" ([sensor_msgs/Image](http://docs.ros.org/api/sensor_msgs/html/msg/Image.html))
```

Comment:
The topic_root parameter can be replaced by correct usage of the namespace of the node I think. I will try to setup some example launch file, so that you can get what I mean. :smiley: